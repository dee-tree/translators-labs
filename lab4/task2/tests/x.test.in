a b c d aa bb cc dd
hello

aaa aaa aa xxx
xxxx  elf lololoshka
anymore

x x x skip
alpha