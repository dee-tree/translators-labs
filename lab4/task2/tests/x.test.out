alpha
anymore
hello
lololoshka
skip
xxxx
